rclcpp/include/rclcpp/service.hpp

```diff
@@ -486,6 +486,14 @@ class Service
   {
     rcl_ret_t ret = rcl_send_response(get_service_handle().get(), &req_id, &response);
 
+    if (ret == RCL_RET_TIMEOUT) {
+      RCLCPP_WARN(
+        node_logger_.get_child("rclcpp"),
```

Comment:
i think "rclcpp" is intentionally added here that tells this log comes from the system.